### PR TITLE
Pomodoro Timer Website using React

### PR DESCRIPTION
**Pull request by VanshikaSabharwal**
### Issue Title:  Pomodoro Timer Website using React JS 

- **Name of the project** : pomodoro-timer
- **Category:**  React
- **Name:** Vanshika Sabharwal
- **GitHub ID:**(https://github.com/VanshikaSabharwal)
- **Idenitfy yourself:**  GSSOC contributor

Closes: #55

### Describe the add-ons or changes you've made 📃

I have added React project pomodoro timer website to the project. 
This project has features like user-friendly interface, responsiveness, no extra stuff to distract user, clean code with comments wherever needed.

Screenshot 2024-05-12 194542

Screenshot 2024-05-12 194609

### CODE REVIEW

File: React-JS-Projects/Basic/pomodoro-timer/README.md
Comment (Avdhesh-Varshney) [inline React-JS-Projects/Basic/pomodoro-timer/README.md:1]: Follow the readme template as mentioned in guidelines.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@VanshikaSabharwal Go through guidelines. If you have any queries, go through other merged prs and their requested changes.

File: React-JS-Projects/Basic/pomodoro-timer/README.md
Comment (VanshikaSabharwal) [reply React-JS-Projects/Basic/pomodoro-timer/README.md:1]: I want to ask why did you removed level tag ? Will this issue not count?

**Comment by VanshikaSabharwal** [comment]
I have added README.md file . So, can you please merge the project .

**Comment by VanshikaSabharwal** [comment]
@Avdhesh-Varshney Can you please tell me what else must be added in the project. So that i can make the necessary changes for merge.

**Review by Avdhesh-Varshney — CHANGES_REQUESTED** [review]
@VanshikaSabharwal Still you have not read the guidelines carefully [Link](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md)

**Comment by VanshikaSabharwal** [comment]
> @VanshikaSabharwal Still you have not read the guidelines carefully [Link](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md)



> @VanshikaSabharwal Still you have not read the guidelines carefully [Link](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md)

Okay, I'll do it

**Comment by VanshikaSabharwal** [comment]
@Avdhesh-Varshney  Can you please add gssoc tag and level tag in small letter to this issue

**Comment by VanshikaSabharwal** [comment]
I made changes @Avdhesh-Varshney in react.json and README.md file.

**Comment by VanshikaSabharwal** [comment]
Also can you please add gssoc and level tag to this issue.

**Comment by Avdhesh-Varshney** [comment]
> Also can you please add gssoc and level tag to this issue.

@VanshikaSabharwal labels will be add automatically when PR will be merged. If you have any query, tag me on discord i will response whenever possible.

**Comment by VanshikaSabharwal** [comment]
Is there any changes to be made in the project . I have made all possible changes according to the guideliness.

## Pull Request for WebMasterLog 💡

### Issue Title: Pomodoro Timer Website using React 

- **Name of the project** : Pomodoro-Timer
- **Category:**  React
- **Name:** Vanshika
- **GitHub ID:** VanshikaSabharwal
- **Idenitfy yourself:** GSSOC




Closes: #55


### Describe the add-ons or changes you've made 📃
I have made a react project -> Pomodoro-Timer.
I have updated the README.md file of project, react.json file in database as well as i have made a README.md file for my Pomodor-Timer .
Also there is proper responsiveness in the Pomodoro-Timer and a 5 minute break after 25 minute work.


### Type of change ☑️

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested? ⚙️




### Checklist: ☑️

- [ ] My code follows the [Contributing Guidelines](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CONTRIBUTING.md) & [Code of Conduct](https://github.com/Avdhesh-Varshney/WebMasterLog/blob/main/CODE_OF_CONDUCT.md) of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly wherever it was hard to understand.
- [ ] My changes generate no new warnings.


### Screenshots 📷

Screenshot 2024-05-12 194542
143436704/b3349092-aeac-4941-b701-e06208f99063)



### Note to reviewers 📄

**Review by Avdhesh-Varshney — COMMENTED** [review]
@VanshikaSabharwal this pr is too much messy and lots of un-necessary commits. So, create a new pr.
- i think screenshot is blank as 0 byte is mentioned. Properly attach after self verifying your screenshot and code.
- Don't update un-necessary code as done in `README.md` by deleting lines.
- name of directory is not `pomodoro-timer`. It's `Pomodoro-Timer` as mentioned in guidelines with proper example.
- No readme file is present to explain the project. That's why i told you to check other successfully merged PRs.

For further queries and doubts, join discord server [link](https://discord.gg/tSqtvHUJzE). Don't mess up the your PR. It will be useful for you in future for showcasing.

**Pull request closed without merging**